Euler_discret.q

```diff
@@ -0,0 +1,49 @@
+\l qlib.q
+\l transform.q
+.import.module `rlang
+@[system; "p 5000"; {-2 x;}]
+// Wiener function
+Wiener: {[t]
+	Z:: .tr.polar ct: -1+count t;

```

Comment:
Bad habit to define a global variable. This is called a function with side effect and should be avoided.